resources/views/pages/products.ejs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% include ../partials/head %>
+    <link rel="stylesheet" href="/resources/css/products.css" />
+  </head>
+  <body class="container">
+    <header>
+      <% include ../partials/header %>
+    </header>
+    <main class="flex-row">
+      <div class="category-list">
+        <% for(var i=0; i < categories.length; i++) { %>
+        <a href="/products/<%= categories[i].key %>"><h1 class="category-list__name"><%= categories[i].name %></h1></a>
+        <% } %>
+      </div>
+      <div class="product-list">
+        <div class="mob-category-list-container">
+        <select class="mob-category-list" onchange="window.location = this.value;">
+        <% for(var i=0; i < categories.length; i++) { %>
+            <% if (categoryId && categoryId===categories[i].id) { %>
+                <option value="/products/<%= categories[i].key %>" selected>
+            <% } else { %>
+                <option value="/products/<%= categories[i].key %>">
+            <% } %>
+            <%= categories[i].name %>
+        </option>
+        <% } %>
+        </select>
+        </div>
+        <% for(var i=0; i < products.length;i) { %>
```

Comment:
Infinite for loop. i not being incremented.

Reply:
its done.